src/forms/upload/camera-capture/camera-capture.controller.js

```
@@ -30,9 +30,16 @@ class CameraCaptureController {
     this.overlaySquareLength = 0;
     this.sensorWidth = 0;
 
-    if (!this.hasGetUserMedia()) {
-      // TODO: haoyuan how to handle get user media not being available?
-      this.$log.warn('getUserMedia() is not supported by your browser');
+    if (this.$window.navigator.mediaDevices === undefined) {
+      this.$log.error('navigator.mediaDevices not accessible on this browser');
```

Deliberately want to surface these errors up to Rollbar for a while. We have some ideas for how this might be fixed in future, but the browser stats from Rollbar will give us more data on what browsers get affected.